--- vmaf-1.5.3/libvmaf/src/feature/adm_tools.c.orig	2020-08-25 01:35:41.000000000 +0200
+++ vmaf-1.5.3/libvmaf/src/feature/adm_tools.c	2021-01-12 18:58:18.525832313 +0100
@@ -35,8 +35,7 @@
 #define MIN(x, y) (((x) < (y)) ? (x) : (y))
 #define MAX(x, y) (((x) > (y)) ? (x) : (y))
 
-#if ARCH_X86
-#ifdef ADM_OPT_RECIP_DIVISION
+#if ARCH_X86 && defined(ADM_OPT_RECIP_DIVISION)
 
 #include <emmintrin.h>
 
@@ -47,7 +46,6 @@
 }
 
 #define DIVS(n, d) ((n) * rcp_s(d))
-#endif //ADM_OPT_RECIP_DIVISION
 #else
 #define DIVS(n, d) ((n) / (d))
 #endif //ARCH_X86
